### ISSUE TITLE
How to use Keras.backend in TensorflowJS ?

### ISSUE THREAD

**Issue by jackfrost1411** [tensorflow/tfjs#4837]
The below code snippet is written in python Keras. 
```
from keras import backend as K
...

layer_name ='Mask'
NewInput = model.get_layer(layer_name).output
visualization = K.function([model.input], [NewInput])
...
```
I want to write this code in Tensorflow.JS .. I am unable to find the right lines in TensorflowJS for this code snippet. 

Can anyone help ?!

**Comment by jackfrost1411** [comment]
I had to reopen this issue as I couldn't find the solution anywhere.. Is there an alternative for doing the above task?

**Comment by pyu10055** [comment]
There some of the concept of backend in the tfjs-core package, what is the particular thing you want to do on the keras backend package?

**Comment by jackfrost1411** [comment]
> There some of the concept of backend in the tfjs-core package, what is the particular thing you want to do on the keras backend package?

I have trained a model Keras that ends with a softmax layer. But I want to get output from an intermediate Convolution layer. How can I do this?

**Comment by pyu10055** [comment]
You can access layers using tfjs-layers function model, example is here
https://stackoverflow.com/questions/50942677/tensorflowjs-how-to-get-inner-layer-output-in-a-cnn-prediction/51316516#51316516
